src/main/java/lol/ui/Sprites.java

```diff
@@ -28,19 +39,37 @@ public ImageView grass() {
     return makeView(grassImage);
   }
 
-  public ImageView archer() {
-    return makeView(archerImage);
-  }
+  public ImageView championsView(String name, int teamID) {
+    if(name.equals("Archer")){
```

Comment:
wouldn't it be better to just have `return makeView(archerImages[teamID]);`?
But ok if you don't want to change that.